### ISSUE TITLE
Release v2.0.0 todos

### ISSUE THREAD

**Issue by jmattheis** [gotify/server#132]
- [ ] update build docker images to go 1.12 gotify/build
  - blocked because https://hub.docker.com/_/golang/ isn't updated yet
- [ ] Update GO_VERSION in gotify/server to 1.12
- [ ] Update and merge https://github.com/gotify/website/pull/6
- [ ] Adjust links in README
- [ ] Testing
  - [ ] Plugin Stuff
  - [ ] WebUI
  - [ ] Swagger UI
  - [ ] CLI
- [ ] Increment API Version in https://github.com/gotify/server/blob/master/docs/package.go#L19
- [ ] Build gotify/server release
- [ ] Update https://github.com/gotify/go-api-client
- [ ] Update https://github.com/gotify/plugin-template

**Comment by eternal-flame-AD** [comment]
> blocked because gimme doesn't support 1.12.0 yet

You need to specify `1.12` instead of `1.12.0`. Gimme matches the version as shown in the [release page](https://golang.org/dl/)

**Comment by jmattheis** [comment]
@eternal-flame-AD Yeah, got it working yesterday too (https://travis-ci.org/gotify/server/builds/500002281#L480) it sucks that it has different versioning than the docker images :(.

**Comment by eternal-flame-AD** [comment]
Yeah, that was unexpected, they used to follow the release versioning, but not this time. Maybe we would need to make GO_VERSION a json file which includes both the docker tag and the release version string, and use `jq` to obtain the result.

**Comment by jmattheis** [comment]
I guess it was always like this, in Docker 1.11 is actually the most recent 1.11 version so 1.11.5. With this we just need to omit the last `.0` from the version when using gimme.

**Comment by eternal-flame-AD** [comment]
It wasn't like this in 1.10. Never mind, maybe they changed their mind in naming patterns for tags. We can strip or add the trailing  .0 manually.

**Comment by jmattheis** [comment]
@eternal-flame-AD 
I've thought of gotify/plugins like a metadata plugins repository, now I think it is a little overkill. I want a place where plugins and also other stuff like chrome extensions could be listed. It should be like an awesome list. What do you think of an awesome list like repo for gotify could be gotify/contrib where plugins,chrome extensions, tutorials, etc could be listed?

**Comment by eternal-flame-AD** [comment]
That would be nice. Should it has its own website, or just a compiled list as a Markdown file?

**Comment by jmattheis** [comment]
I'm unsure, I guess we start with a markdown file and see if there is demand for it. What do you think is better?

**Comment by eternal-flame-AD** [comment]
I also prefer the simplicity of a simple markdown list to a whole website, especially when there are not many items there. We could consider upgrading it to a website after we collected a relatively substantial amount of items.

**Comment by GalvinGao** [comment]
We might want to use a small script-like file to update the `README.md` file when each time the branch receives a push. Some ideas here:
- using `WebHook` which might require an external server
- using `GitHub Actions` but I'm not sure this is the case
- using `Travis-CI`

**Comment by jmattheis** [comment]
@GalvinGao In what context is you comment? If it is something unrelated to the discussion above please open a new issue.

**Comment by GalvinGao** [comment]
@jmattheis I've edited the comment to make it better ;)

**Comment by jmattheis** [comment]
@GalvinGao Sorry I still don't understand it, please clarify what you mean with index website, what readme you refer to and what your idea is trying to accomplish.
